docs/source/configuring.md

```diff
@@ -0,0 +1,145 @@
+---
+title: 'Configuring Rover'
+sidebar_title: 'Configuring'
+description: "Setting up Rover to work exactly how you'd like"
+---
+
+## Profiles
+
+Rover uses configuration profiles to track certain global config values like api 
+keys. These values are associated with a profile, and can be chosen at runtime 
+with the `--profile` flag. By default, the `default` profile is used.
+
+```
+graph check my-company@prod --profile work
+```
+
+Profiles are useful for managing multiple user API keys, but if you don't need 
+to switch profiles often, the `default` profile is used when no `--profile` is 
+passed.
+
+To see all commands for viewing, editing, and deleting profiles, run the 
+following in Rover:
+
+```
+rover config profile --help
+```
+
+## Authenticating with Apollo Studio
+
+For many of Rover's operations, you will not need to authenticate at all!
+
+However, to use Apollo Studio services, such as checks, either in local
+development or in CI, you'll want to grant the CLI permission to perform a set
+of operations on your graphs. You can do this via API tokens.
+
+Currently, there are 2 kinds of tokens you can generate:
+
+| Type | Generatable by | Permissions | Recommended Use |
+|---------|---------------------|-------------------------------|--------------|
+| User | Apollo Studio User | Same as the account | Local development machine |
+| Graph | Apollo Studio Admin | Can only access a single graph | CI |
+
+You can authenticate the CLI via a command or an environment variable, using
+either type of token. 
```

Comment:
I wonder if it might be better to link to another set of Apollo docs that explains this nuance. I imagine Rover is not the only client that needs these API keys (though I could be wrong!)